webrtc_models/__init__.py

```diff
@@ -56,3 +56,4 @@ class RTCIceCandidate(_RTCBaseModel):
     """
 
     candidate: str
+    sdp_m_line_index: int
```

Comment:
Please add an alias so the field is serialized as specified in the spec. See ice_servers above how to do it

Reply:
Ok will do. I haven't made this optional which is different to the spec. If I make it optional I'll probably need to add sdpMid because one or the other is required. What do you think?

Reply:
We should be spec conform

Reply:
Ok that's done